docs/UserGuide.md

```diff
@@ -239,6 +213,25 @@ Edits the information of a particular company.
 **Expected UI** <br/>
 ![EditUI.png](EditUI.png)
 
+### Filter companies by application status: `filter` ###
+Can't remember which applications you haven't submitted or want to prepare for the companies that have offered you an interview? 
+Filter the list of companies by their application status using the `filter` command!
+
+**Format:** `filter s/APPLICATION_STATUS`
+
+**Examples:**
+* `list` followed by`filter s/PI` filters the list of companies to show only companies with status "Pending Interview".
+
+**Note:**
+- APPLICATION_STATUS should be one of the following: `PA`, `PI`, `PO`, `A`, `R`.
+
```

Comment:
Once again, should we mention about case sensitivity?

Reply:
yup, will add it in!